File: Coding-Interviews/17-打印从1到最大的n位数.cpp
#include <string>
#include <vector>
using namespace std;
class Solution
{
public:
    vector<int> printNumbers(int n)
    {
        string      max_num(n, '9');
        vector<int> ret;
        for (int i = 1; i <= stoi(max_num); i++)
            ret.push_back(i);
        return ret;
    }
};

File: Coding-Interviews/17-打印从1到最大的n位数.py
from math import pow
from typing import List


class Solution:
    def printNumbers(self, n: int) -> List[int]:
        max = 0
        for i in range(0, n):
            max += 9*int(pow(10, i))
        res = [i for i in range(1, max+1)]
        return res
